### ISSUE TITLE
@if display(selector) causing syntax error on latest branch

### ISSUE THREAD

**Issue by bob2517** [Active-CSS/active-css#206]
Spotted this the other day. Not sure how far-reaching this bug is yet. Fixing now.

**Comment by bob2517** [comment]
Repeatable

**Comment by bob2517** [comment]
if-display doesn't appear to use the latest selector method in the core, which is the first thing to fix.

**Comment by bob2517** [comment]
That was actually the issue. It's fixed offline. I'm mid another issue, and this fix will go up once that is done.

**Comment by bob2517** [comment]
Closing in preparation for release.

**Comment by bob2517** [comment]
Closing in preparation for release.
